front/lib/actions/mcp_internal_actions/output_schemas.ts

```diff
@@ -497,6 +498,36 @@ const NotificationImageContentSchema = z.object({
 
 type ImageProgressOutput = z.infer<typeof NotificationImageContentSchema>;
 
+// Schema for the resource of a notification where the tool is asking for tool approval.
+// This schema contains all the information that the MCP server runner
+// needs to emit an event for tool approval.
+const NotificationToolApproveExecutionContentSchema = z.object({
```

Comment:
Out of consistency we need to rename all types to NotificationToolApproveBubbleUpXxxx